Gruntfile.js

```diff
@@ -45,15 +45,18 @@ module.exports = function (grunt) {
     parser.parseString(xmlBuffer, function (err, result) {
       // Limit cities data to Dutch name and zip code. That's all we currently
       // need in the application.
-      cities = result.cdbxml.cities.city.map(
-        function (city) {
-          return {
-            'label': city.zip + ' ' + city.labelnl,
-            'name': city.labelnl,
-            'zip': city.zip
-          };
-        }
-      );
+      cities = result.cdbxml.cities.city.reduce(
+        function (filteredCities, city) {
+          if (city.submunicipality && city.submunicipality === 'true') {
```

Comment:
I think we have to include everything that has the `submunicipality` property/attribute, not only those that have it set to `true`.

Reply:
For example: http://taxonomy.uitdatabank.be/api/city

```
<city id="3020_HERENT" label="Herent" zip="3020" labelnl="Herent" submunicipality="false"/>
<city id="3012_WILSELE" label="Wilsele (Leuven)" zip="3012" labelnl="Wilsele (Leuven)" parent="3000_LEUVEN" submunicipality="true"/>
```
-> Should both be in the list of cities in udb3

```

Reply:
Yes, that's indeed an error 😅